tests/tests-skeletons/check-INTEGER.c

```diff
@@ -133,7 +133,7 @@ check_unsigned(uint8_t *buf, int size, unsigned long check_long, int check_ret)
 	assert(shared_scratch_start < scratch + sizeof(scratch));
 	assert(ret == 0);
 	ret = snprintf(verify, sizeof(verify), "%lu", check_long);
-	assert(ret < sizeof(verify));
+	assert(ret < (int)sizeof(verify));
```

Comment:
`assert(ret >= 0 && (size_t)ret < sizeof(verify));`

Otherwise (int) does narrowing, and not checking <0 doesn't check for system errors.